forgot to add the date field which was a requirement

britt-allen opened the pull request.

britt-allen committed:
forgot to add the date field which was a requirement

ram-kishore-odi reviewed (approved):
The change looks good to me.

ian-r-rose commented on transform/models/marts/engaged_ca/ec__events_form_submit.sql:
```diff
@@ -22,7 +23,7 @@ form_submits as (
         end as page_location_of_event,
         referral_source,
-        total_event_count
+        sum(total_event_count) as total_event_count
```
I'd say this is one of the dangers of using `group by all`: anything that doesn't have an aggregation implicitly becomes part of the group by keys. Maybe a good reason to avoid using it (despite how nice it can be)

britt-allen replied:
i agree, i'll update

ian-r-rose reviewed (approved):
A bit more context here would be helpful: what was wrong about the previous model, and how does this fix it? More is going on here than just adding one new column because of the aggregation

MACKAT05 commented on transform/models/marts/engaged_ca/ec__events_form_submit.sql:
```diff
@@ -4,6 +4,7 @@ with source as (
 
 form_submits as (
     select
+        event_date,
```
don't forget to update the yml 
 https://github.com/cagov/caldata-web-analytics/blob/f3a9edd83734a594c512ef5ef1804832b5993d4a/transform/models/marts/_mart_models.yml#L54

britt-allen replied:
ty!

britt-allen commented:
> A bit more context here would be helpful: what was wrong about the previous model, and how does this fix it? More is going on here than just adding one new column because of the aggregation

The original ask was to create a mart model that showed total event counts of form submits by date and referral source. I added in `page_location_of_event` to add more context. Since the original ask was to include event date and we did not I added it in the PR. I also corrected an aggregation error where I was grouping by `total_event_count` rather than summing it and grouped by the newly added `event_date` and remaining columns that were already a part of the group by. More info on this task can be read [here](https://app.asana.com/1/1202865175765955/project/1210910778445814/task/1210943420222995?focus=true).

britt-allen committed:
fixes related to review feedback
